SynthSeg/metrics_model.py

```diff
@@ -113,7 +114,7 @@ def get_config(self):
         }
         return config
 
-    def __call__(self, gt, pred):
+    def __call__(self, gt, pred, **kwargs):
```

Comment:
Out of curiosity, do you know what these `kwargs` are?

Reply:
It's "sample_weight" (https://github.com/keras-team/keras/blob/601488fd4c1468ae7872e132e0f1c9843df54182/keras/losses.py#L102C40-L102C53). I thought of using **kwargs for future version compatibility in case it changes in tf. Do you think we can keep it or change it to "sample_weight"?

Reply:
kwargs is perfect, I was just curious :)